census/src/test/scala/grasshopper/census/util/TestData.scala

```diff
@@ -22,6 +22,16 @@ object TestData {
     fjson.parseJson.convertTo[Feature]
   }
 
+  def getTigerLine4(): Feature = {
```

Comment:
I'm guessing were going to be adding more and more TIGER line test cases.  Probably worth just reading in and loading all files from a given directory.  Also, probably worth naming the files something related to the test data instead of just 1, 2, x.  Not something we need to fix for this PR, though.